notebooks/combine_datasets.ipynb

```diff
@@ -30,22 +30,9 @@
   },
```

Comment:
<p>Why are we removing all notebook outputs? You mentioned "latency issues," but I don't understand quite what you mean. I think having inline charts rendered right in the file on github is one of the huge draws of notebooks. Without the charts, this notebook doesn't tell its story until you go through the effort of cloning, installing jupyter, and running it yourself.  I guess I just want to know what your issue is, in case there's another solution that lets us keep all the charts rendered.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/srensi/tmprss2/pull/16/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
I just find it a little annoying when I accidentally open up a notebook, and then having it as a diff in the commit. And sometimes, with the notebooks using plt plots, the size of the notebook can become  > couple MBs which does introduce silly awaiting whenever there's a small difference to a notebook. Purely personal tho, and I don't mind either way at the moment (until it becomes annoting).